apps/institution-ms/app/src/main/docs/openapi.json

```diff
@@ -797,6 +797,15 @@
           "schema" : {
             "type" : "string"
           }
+        }, {
+          "name" : "subunits",
```

Comment:
since this is a boolean param, maybe we can name it more explicitly, something like "enable...", "allow", "is..." should suffice :)